From 290de2550737a8d1576099c873ede0c7f96dacb2 Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@gentoo.org>
Date: Sat, 18 Aug 2012 23:39:48 -0500
Subject: [PATCH] Fixes for using glib 2.32

With glib 2.32, gmodule is a separate module, and while there are some fixes in mate-keyring already, they AC_SUBST GLIB_LIBS instead of the proper GMODULE_LIBS.

Signed-off-by: Steev Klimaszewski <steev@rose.hackershack.net>
---
 configure.in    |    2 +-
 gck/Makefile.am |    1 +
 2 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index a7a9f4e..78b8982 100644
--- a/configure.in
+++ b/configure.in
@@ -71,7 +71,7 @@ AC_SUBST(GLIB_LIBS)
 
 PKG_CHECK_MODULES(GMODULE, gmodule-no-export-2.0)
 AC_SUBST(GLIB_CFLAGS)
-AC_SUBST(GLIB_LIBS)
+AC_SUBST(GMODULE_LIBS)
 
 PKG_CHECK_MODULES(GTHREAD, gthread-2.0 >= 2.8.0)
 AC_SUBST(GTHREAD_CFLAGS)
diff --git a/gck/Makefile.am b/gck/Makefile.am
index 0eab2cb..f759d42 100644
--- a/gck/Makefile.am
+++ b/gck/Makefile.am
@@ -57,6 +57,7 @@ libmategck_la_LIBADD = \
 	$(GOBJECT_LIBS) \
 	$(GTHREAD_LIBS) \
 	$(GIO_LIBS) \
+	$(GMODULE_LIBS) \
 	$(GLIB_LIBS)
 
 gck-marshal.h: gck-marshal.list $(GLIB_GENMARSHAL)
-- 
1.7.3.4

